examples/attribute_inference_example.py

```diff
@@ -17,7 +21,7 @@
 
 from aisdc.attacks import attribute_attack  # pylint: disable = import-error
 from aisdc.attacks.attack_report_formatter import GenerateJSONModule
-from aisdc.attacks.target import Target
+from aisdc.attacks.target import Target  # pylint: disable = import-error
```

Comment:
out of curiosity -- why don't we have an import error that needs to be disabled on line 23?

Reply:
I did check previously and it has some issues with linting. Anyways double checked this now and it was working fine so added disabling functionality.